src/app/views/record/record.routes.js

```diff
@@ -21,12 +21,17 @@ function getStates() {
               $window.location.href = `https://ezid.cdlib.org/id/${bcid}`;
               return false;
             }
-
             return RecordService.get(bcid.trim())
               .then(response => {
-                if (response.status === 204) {
-                  $state.go('notFound', { path: '404' });
-                }
+		// If the response.status is anything other than 200,
+		//  it is most likely that the it is  a 403 error (forbidden)
+		// or the resource does not exist.  It would be better to trap by 
+		// specific status codes, however, i've found that status codes get
+		// mangled when exceptions occurr, so in this case, we politely 
+		// fail using the 404 route
+                if (response.status != 200) {
```

Comment:
anything > then 299 should not appear here, but in the catch below.

I did fix a bug in the api where `{projectId: 1}` was returned if the root identifier existed, but the suffix was invalid. It now returns null, just as if the root identifier doesn't exists.

The api already returns a 403 if the user is trying to access data in an expedition they don't have access to.

Can you give me an example of a record that returned a "mangled" status code? I'd rather fix this on the api side

Reply:
won't fix for this PR